### DISCUSSION TITLE
read JSON file uisng JSVM

### DISCUSSION THREAD

**Discussion by wiezmankimchi** [pocketbase/pocketbase#5961] [Q&A]
Trying to import a JSON file as a source for seeding a collection, cannot read the JSON file.
Is there a way to read the file as JSON and not a "string"?
is there a way to use JSON.parse(file), or use another JS library to do so?

**Comment by ganigeorgiev** [comment]
If it is a valid JSON file then requiring it should work:

```js
const data = require("/path/to/your/file.json")
```
_(if you are using relative paths, make sure to check https://pocketbase.io/docs/js-overview/#relative-paths)_
